src/t8_element_cxx.hxx

```diff
@@ -74,6 +74,7 @@ public:
 
   /** Returns true, if there is one element in the tree, that does not refine into 2^dim children.
    * Returns false otherwise.
+   * \return                    non-zero if there is one element in the tree that does not refine into 2^dim children.
```

Comment:
We should document that this function is MPI collective.
Calling this function only on one process will cause a deadlock.